### DISCUSSION TITLE
Private endpoint support

### DISCUSSION THREAD

**Discussion by sikkiv** [tomkerkhove/promitor#1527] [General] [answered]
Hi, 

I am wondering will there be or is there already support for Private Endpoints and using Promitor to connect to Azure Monitor via a Private Endpoint?
Because I can't find anything related to that in the documentation or the backlog (or I have maybe overlooked it 😄 )

Best regards

**Comment by tomkerkhove** [comment, marked as answer]
Today this is not supported but a valid request, adding it to the backlog!

I'm new to private endpoint though, do you have some good docs for Azure Monitor + Private endpoint?

**Reply by tomkerkhove** [reply]
https://github.com/tomkerkhove/promitor/issues/1530

**Reply by sikkiv** [reply]
Hey :) 
that is some fantastic news!

Here is the official documentation for the Private endpoint connection and Azure Monitor: https://docs.microsoft.com/en-us/azure/azure-monitor/logs/private-link-security

Thanks for your quick reply and looking forward to the next release ;)
